File: RNA.fs
#light

let [|_;s|] = System.Environment.GetCommandLineArgs()
let rna = seq { 
  for b in s do 
    if 'T' = b then 
      yield 'U' 
    else 
      yield b 
}
printfn "%s" (new System.String([| for b in rna -> b|]))

File: rna.erl
-module(rna).
-export([transcode/1]).

transcode(Dna) ->
  transcode(Dna,[]).

transcode([],Rna) ->
  Rna;
transcode([$T|Dna],Rna) ->
  transcode(Dna, Rna ++ [$U]);
transcode([B|Dna],Rna) ->
  transcode(Dna, Rna ++ [B]).
